pyproject.toml

```diff
@@ -160,22 +160,13 @@ PKG_CONFIG_PATH="/project/.openblas"
 LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/project/.openblas/lib"
 
 [tool.cibuildwheel.macos]
-# universal2 wheels are not supported (see gh-21233), use `delocate-fuse` if you need them
-archs = "x86_64 arm64"
-test-skip = "*_universal2:arm64"
 # Not clear why the DYLD_LIBRARY_PATH is not passed through from the environment
 repair-wheel-command = [
   "export DYLD_LIBRARY_PATH=$PWD/.openblas/lib",
   "echo DYLD_LIBRARY_PATH $DYLD_LIBRARY_PATH",
   "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}",
 ]
 
-[tool.cibuildwheel.macos.environment]
```

Comment:
these variables are specified in the individual CI configurations. Can restore if necessary.

Reply:
That looks fine to me in principle; better to have them defined where they are used. The one oddity is that `RUNNER_OS` is now defined in this file for Linux and not for Windows/macOS - but that was already looking a bit odd, so if it works like this, no problem.